src/main/java/org/springframework/data/tarantool/repository/Query.java

```diff
@@ -25,4 +26,12 @@
      * @return the callable function name
      */
     String function();
+
+    /**
+     * Specify the response structure that the connector will expect from Tarantool by this parameter,
+     * the connector will use a certain stack of converters to translate the incoming result into a class object
```

Comment:
This phrase is redundant, speak of a use case here, not about the implementation details.

Reply:
Rewrite